conf.d/virtualfish.fish

```diff
@@ -1,12 +1,10 @@
-if set -q $VIRTUALFISH_PYTHON
-    set VIRTUALFISH_PYTHON python
-end
+set -q VIRTUALFISH_PYTHON || set VIRTUALFISH_PYTHON python
```

Comment:
If `VIRTUALFISH_PYTHON` is set, then the second part is never ran.

Reply:
Note: `||` restricts this to fish >= 3.0.

Officially this plugin supports fish >= 2.2.0. Just use `or`:

```fish
set -q VIRTUALFISH_PYTHON
or set VIRTUALFISH_PYTHON python
```

(or keep the `if`)

Reply:
```suggestion
set -q VIRTUALFISH_PYTHON
or set VIRTUALFISH_PYTHON python
```